themes/powerline/powerline.theme

```diff
@@ -304,10 +305,19 @@ __powerline_path_split() {  # path_format, [separator_format], [last_dir_format]
 # __powerline_path_split(), which forgets the old colors for the arrow. This
 # function lets the arrow go first.
 __powerline_path_section() {
-    local powerline_path section_arrow
+    local powerline_path section_arrow dirstack_indicator
     __powerline_section_arrow "${POWERLINE_PATH_COLOR[@]}"
     __powerline_path_split
-    powerline_sections+=${section_arrow}${powerline_path}
+    __powerline_dirstack_indicator
+    powerline_sections+="${section_arrow}${powerline_path}${dirstack_indicator}"
```

Comment:
I do like how it looks, but _everything_ in powerline is its own section. I like the placement, but I think it should be its own section right after the path. I think the `POWERLINE_NEUTRAL_COLOR` would look good right next to the path actually, so no need to add a new color for it.

Or maybe it should have its own color, some might want to customize it. But we could default it to the neutral color:
```bash
[[ -z ${POWERLINE_DIRSTACK_COLOR[@]+x}                ]] && POWERLINE_DIRSTACK_COLOR=("${POWERLINE_NEUTRAL_COLOR[@]}")
```
that line would obviously have to come after line 129.